File: Anyelo Carrillo Java/Interaccion usuario.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // Crear un objeto Scanner para leer la entrada del usuario
        Scanner scanner = new Scanner(System.in);

        // Solicitar y leer los nombres completos
        System.out.print("Escriba sus nombres completos: ");
        String a = scanner.nextLine();

        // Solicitar y leer los apellidos completos
        System.out.print("Escriba sus apellidos completos: ");
        String b = scanner.nextLine();

        // Solicitar y leer la profesión
        System.out.print("Escriba su profesión: ");
        String c = scanner.nextLine();

        // Solicitar y leer el año de nacimiento
        System.out.print("Escriba su año de nacimiento: ");
        int d = scanner.nextInt();

        // Calcular la edad en el año 2025
        int e = 2025 - d;

        // Mostrar el resultado
        System.out.println("El (La) " + c + " " + a + " " + b + " tiene " + e + " años");

        // Cerrar el Scanner
        scanner.close();
    }
}

File: Anyelo Carrillo Go/Interaccion usuario.go
package main

import (
	"fmt"
)

func main() {
	// Solicitar al usuario que ingrese sus nombres completos
	fmt.Print("Escriba sus nombres completos: ")
	var a string
	fmt.Scanln(&a)

	// Solicitar al usuario que ingrese sus apellidos completos
	fmt.Print("Escriba sus apellidos completos: ")
	var b string
	fmt.Scanln(&b)

	// Solicitar al usuario que ingrese su profesión
	fmt.Print("Escriba su profesión: ")
	var c string
	fmt.Scanln(&c)

	// Solicitar al usuario que ingrese su año de nacimiento
	fmt.Print("Escriba su año de nacimiento: ")
	var d int
	fmt.Scanln(&d)

	// Calcular la edad en el año 2025
	e := 2025 - d

	// Mostrar el resultado
	fmt.Printf("El (La) %s %s %s tiene %d años\n", c, a, b, e)
}
